workers/unity/Packages/com.improbable.gdk.core/Worker/ConnectionHandlers/SpatialOSConnectionHandler.cs

```diff
@@ -0,0 +1,52 @@
+using Improbable.Worker.CInterop;
+
+namespace Improbable.Gdk.Core
+{
+    public class SpatialOSConnectionHandler : IConnectionHandler
+    {
+        private readonly ViewDiff diff = new ViewDiff();
+        private readonly OpListDeserializer deserializer = new OpListDeserializer();
+        private readonly SerializedMessagesToSend messagesToSend = new SerializedMessagesToSend();
```

Comment:
For clarity can we rename this to `serializedMessagesToSend`?

Reply:
yep